lib/sqlalchemy/dialects/mssql/base.py

```diff
@@ -2590,12 +2591,19 @@ def get_columns(self, connection, tablename, dbname, owner, schema, **kw):
                 columns.c.table_name == tablename,
                 columns.c.table_schema == owner,
             )
+            table_fullname = "%s.%s" % (owner, tablename)
         else:
             whereclause = columns.c.table_name == tablename
+            table_fullname = tablename
         s = sql.select(
             [columns], whereclause, order_by=[columns.c.ordinal_position]
         )
 
+        cc = ischema.computed_columns
```

Comment:
do we need to be concerned about this table not being present in some versions of SQL server, like older versions, or azure / cloud versions?

Reply:
it seems so regarding azure-cloud. I'm not sure regarding older versions.
https://docs.microsoft.com/en-us/sql/relational-databases/system-catalog-views/sys-computed-columns-transact-sql?view=sql-server-ver15

Reply:
I've found this comment, from 2009 regarding `computed_columns`, so it should be fairly old

https://stackoverflow.com/questions/1484147/get-list-of-computed-columns-in-database-table-sql-server

Reply:
here another mssql 2005+ 
https://stackoverflow.com/questions/5047099/sql-server-computed-column-defintions-retrievable-from-database

Reply:
ok then let's keep it

Reply:
We could wrap it in a try catch just to be sure

Reply:
those try/catches are themselves a lot of work to get right and test and if this has been there since SQL server 2005 I have a feeling we dont need to worry about it

Reply:
if I go to MS's docs and look at "sys.tables", which is definitely everywhere, it lists the same four green checkboxes as sys.computed_columns, so this is good as it is.